common/national_focus/Cuba.txt

```diff
@@ -17,7 +17,7 @@ focus_tree = {
 
 	focus = { #POLITICAL Tree Start
 		id = CUB_results_of_the_presidential_election
-		icon = GFX_goal_cuba_presidential_election
```

Comment:
This icon doesn't seem very style-consistent, but it definitely seems better than a generic, in my opinion